.github/workflows/qat.yml

```
@@ -15,160 +15,70 @@ on:
-  CHECKLY_REPO_BRANCH: ${{ github.head_ref }}
-  CHECKLY_REPO_COMMIT_OWNER: ${{ github.actor }}
-  CHECKLY_REPO_SHA: ${{ github.sha }}
-  CHECKLY_TEST_ENVIRONMENT: ${{ inputs.QA_SITE_DOMAIN }}
-  FEATURE_BRANCH_NAME: $(echo "${{ github.head_ref || github.ref_name }}" | sed -e 's/feature\/\(.*\)/\1/' | tr '[:upper:]' '[:lower:]')
-  NPM_QAT_SCRIPT: "qat"
-
 jobs:
-  check-qat:
-    name: Check for QAT
+  qat:
+    name: QAT
     runs-on: ubuntu-latest
     timeout-minutes: 90
     if: ${{ github.repository != 'penske-media-corp/github-workflows-wordpress' }}
 
     concurrency:
       group: ${{ github.ref_name || github.run_id }}-qat-check
       cancel-in-progress: true
-
+    
     steps:
-      - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0 # Check out the full git history
-      - name: Check for Checkly Config
-        id: check_files
-        uses: andstor/file-existence-action@v3
-        with:
-          files: "**/checkly.config.ts"
-
-      - name: Checkly Config Found
-        id: file_exists
-        if: steps.check_files.outputs.files_exists == 'true'
-        # Only runs if all of the files exists
-        run: echo Checkly config found
-
-      # Poll the PMCQA environment to wait for a deployment triggered by Jenkins to complete before continuing the workflow.
-      # This ensures any later steps that interact with PMCQA reflect the code changes from the pull request rather than an older version.
-      - name: Check PMCQA
-        if: ${{ inputs.THEME_NAME  != '' && inputs.QA_SITE_DOMAIN != '' && steps.check_files.outputs.files_exists == 'true' && github.ref != 'refs/heads/main'}}
-        timeout-minutes: 10
-        run: |
-          # `version-hash` is created by jenkins script when a branch is pushed
-          # https://github.com/penske-media-corp/pmc-jenkins-scripts/blob/8ee0244a8485b61cfa6d3f9af00f7e6a55ddb042/bin/push-pmcqa-repo.sh#L36-L38
-          hash_url="https://${{ env.FEATURE_BRANCH_NAME }}.${{ inputs.QA_SITE_DOMAIN }}/wp-content-vipgo-sites/${{ env.FEATURE_BRANCH_NAME }}/themes/vip/${{ inputs.THEME_NAME }}/version-hash"
-          echo "HASH_URL: $hash_url"
-          hash=`curl --no-progress-meter --insecure $hash_url`
-          until [[ $hash == ${{ github.event.pull_request.head.sha }} ]]; do
-              echo -n "Waiting for Jenkins Deployment ... \n"
-              hash=`curl --no-progress-meter --insecure $hash_url`
-              sleep 5
-          done
-    outputs:
-      status: ${{ steps.check_files.outputs.files_exists }}
-
-  checkly-deploy:
-    name: Checkly Run and Deploy
-    runs-on: ubuntu-latest
-    timeout-minutes: 90
-    # if the check_files step in the check-qat job doesn't fail
-    needs:
-      - check-qat
-    if: needs.check-qat.outputs.status == 'true'
-
-    concurrency:
-      group: ${{ github.ref_name || github.run_id }}-checkly-qat
-      cancel-in-progress: true
+      - name: Check out code
+        uses: actions/checkout@v3
```

No reason to use an old version.

```suggestion
      - name: Check out code
        uses: actions/checkout@v4
```